portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/fileinstall/OSGIConfiguration.testcase

```diff
@@ -18,6 +18,13 @@ definition {
 		if (${testPortalInstance} == "true") {
 			PortalInstances.tearDownCP();
 		}
```

Comment:
This is fine, but you should pull the cleanup outside of the else statement (testPortalInstance is set to true in CI runs or specific local runs, so this wont be cleaned up well in CI runs)